### PR TITLE
feat: port contracts periphery to contracts bedrock

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Moves all of the contracts to a single repo so that the same tooling and standards can be applied to them. This PR migrates `contracts-periphery` to `contracts-bedrock`, including all of the tests.

### CODE REVIEW

File: packages/sdk/src/cross-chain-messenger.ts [hunk 1]
```diff
@@ -409,7 +409,7 @@ export class CrossChainMessenger {
       updated.message
     )
 
-    // EVERYTHING following here is basically repeating the logic from getMessagesByTransaction 
+    // EVERYTHING following here is basically repeating the logic from getMessagesByTransaction
```
Comment (roninjin10) [inline packages/sdk/src/cross-chain-messenger.ts]: This is in develop already if you rebase

**Review by refcell — APPROVED** [review]
LGTM

File: .changeset/beige-masks-prove.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@eth-optimism/chain-mon': patch
```
Comment (roninjin10) [inline .changeset/beige-masks-prove.md:2]:
```suggestion
'@eth-optimism/chain-mon': patch
'@eth-optimism/core-utils': patch
```

**Review by roninjin10 — APPROVED** [review]

**Comment by tynes** [comment]
Thanks for rebasing this @clabby

File: .vscode/settings.json [hunk 3]
```diff
@@ -33,4 +29,4 @@
   "eslint.format.enable": true,
   "editorconfig.generateAuto": false,
   "files.trimTrailingWhitespace": true
-}
\ No newline at end of file
+}
```
Comment (maurelian) [inline .vscode/settings.json]: Did something format this automatically? 
I [recently removed](https://github.com/ethereum-optimism/optimism/commit/0249d87936317d2b91f649bf318e08779bb92d1b) the new line at EOF. 

It'd be helpful to me if we can not make this change.

File: op-bindings/bindings/erc20_more.go
Comment (maurelian) [inline op-bindings/bindings/erc20_more.go:1]: Why would this change, and do we need this binding?

Comment (tynes) [reply op-bindings/bindings/erc20_more.go:1]: There is more than 1 erc20 impl included, it deterministically selects which one and now the other one is selected

File: .vscode/settings.json (hunk 3 above)
Comment (tynes) [reply .vscode/settings.json]: This was done automatically, can revert, seems like different editors have different preferences

Comment (maurelian) [reply .vscode/settings.json]: Appreciate it, I've struggled to find the setting to make it not happen.

File: packages/contracts-periphery/config/drippie/ethereum.ts
Comment (maurelian) [inline packages/contracts-periphery/config/drippie/ethereum.ts:1]: what replaces this?

File: .vscode/settings.json (hunk 3 above)
Comment (tynes) [inline .vscode/settings.json]: Still need to do this

File: packages/contracts-bedrock/contracts/test/Helpers.sol
Comment (maurelian) [inline packages/contracts-bedrock/contracts/test/Helpers.sol:1]: weird that this is an addition? And I'm not seeing it in the periphery package on develop.

I'm going to assume it was accidentally deleted somehow and let it go.

**Review by maurelian — APPROVED** [review]
Let's get that money

File: packages/contracts-bedrock/contracts/test/Helpers.sol
Comment (tynes) [reply packages/contracts-bedrock/contracts/test/Helpers.sol:1]: these were a bunch of helpers in contracts periphery consolidated

**Pull request merged**